### PR TITLE
BK-2061 New Book Reader for Booktype

### PR DESCRIPTION
**Pull request by olegpshenichniy**
Hi @eos87 ,

During instance update process, we must check (and merge if needed) files which are changed in this PR and  overloaded inside instance.

### CODE REVIEW

File: lib/booktype/apps/reader/templates/reader/book_full_view.html [hunk 1]
```diff
@@ -3,69 +3,90 @@
...
-<div class="container">
-    <div class="row two-col">
-        <div class="col-xs-12">
-            <div class="box white table_of_contents">
-                <h2 class="box-title">{% trans "Table of Contents" %}</h2>
+  <!-- Bookbar -->
+  <div class="bookbar">
+    <h1>
+      <a class="draft-stat" href="{% url 'reader:infopage' book.url_title %}">{% trans "Info" %}</a>
+      <span class="draft-stat toctoggle closed" id="toctoggle">
```
Comment (eos87) [inline lib/booktype/apps/reader/templates/reader/book_full_view.html]: I would change this for an `a` tag or enable `cursor: pointer` to give it more a button feeling :)

File: lib/booktype/apps/reader/templates/reader/book_info_page.html [hunk 2]
```diff
@@ -49,16 +49,24 @@ <h2 class="box-title">{{ book.title }}</h2>
...
-                   {% check_perm 'edit.edit_book' %}
+                    {% check_perm 'edit.edit_book' %}
                     <button data-href="{% url 'edit:editor' book.url_title %}" role="button" class="btn btn-default float-left" rel="tooltip" data-placement="bottom" data-original-title="{% trans 'Click this button to open the editor at the Table of Contents' %}">
                         {% trans "Edit this Book" %}
                     </button>
                     {% endcheck_perm %}
+
+                    {% check_perm 'reader.can_view_full_page' %}
+                    <button data-href="{% url 'reader:full_view' bookid=book.url_title %}" role="button" class="btn btn-default float-left" rel="tooltip" data-placement="bottom" data-original-title="{% trans 'Click this button to view the draft of the book as a single page' %}">
+                        {% trans "Reader" %}
```
Comment (eos87) [inline lib/booktype/apps/reader/templates/reader/book_info_page.html]: I think we should rename this to "Read Book" or just "Read".

**Comment by eos87** [comment]
One last thing is to update the translation strings :)

**Comment by olegpshenichniy** [comment]
@eos87 , PR was updated. I did not add compiled messages. Will do it in the nest PRs.

**Pull request merged**